src/global/Constants.h

```diff
@@ -220,10 +221,10 @@ inline auto& RuntimeParameters() {
         SizeT<"lazy-index-scan-queue-size">{20},
         SizeT<"lazy-index-scan-num-threads">{10},
         ensureStrictPositivity(
-            StreamableParameter<
-                ad_utility::ParseableDuration<std::chrono::seconds>,
-                "default-query-timeout">{30}),
-        SizeT<"lazy-index-scan-max-size-materialization">{1'000'000}};
+            DurationParameter<std::chrono::seconds, "default-query-timeout">{
+                30}),
```

Comment:
I still like the `30s`. Redundancy is not bad if it increases typesafety and is therefore automatically checked.
Consider if I change this to `milliseconds` at some point, then `30` silently gets `30ms` (rather short), whereas the other one breaks the build (or does the right thing if we add an implicit duration_cast to the constructor of `ParseableDuration`.